match2/wikis/starcraft2/match_legacy.lua

```diff
@@ -119,6 +119,9 @@ function p.convertParameters(match2)
 
 		if match.resulttype == 'default' then
 			match.resulttype = string.upper(match.walkover or '')
+			if match.resulttype == 'L' then
```

Comment:
```suggestion
			if match.resulttype:lower() == 'l' then
```

Reply:
the line before it makes sure it is upper case, why compare it as lower case then?

Reply:
Missed it